centrallix/tests/test_moneytype_obj_internal_BuildBinaryItem_00.c

```diff
@@ -0,0 +1,63 @@
+#include "obj.h"
+#include <assert.h>
+#include <stdio.h>
+#include "expression.h"
+#include "cxlib/xstring.h"
+
+long long
+test(char** name)
+{
+    *name = "moneytype_00 - obj_internal_BuildBinaryItem";
+    char** item;
+    item = (char**)malloc(24);
+    int* itemlen;
+    itemlen = (int*)malloc(4);
+    
+    //Creating an ObjData with a moneytype inside, pObjData for parameters
+    ObjData moneyData;
+    MoneyType unionFiller = {70000};
+    moneyData.Money = &unionFiller;
+    pObjData moneyDataPtr = &moneyData;
+    
+    //Utilizing expression.h functions to initialize expression and paramobjects
+    pExpression testExp = expAllocExpression();
+    pParamObjects testParamObjects = expCreateParamList();
+    
+    //Buffer must be at least 12 bytes
```

Comment:
No it was a comment in his code so I relayed the message here.  I figure it has to do with the range of things that can be passed into it.  Potentially something else is a big enough struct that requires 12 bytes.  Looking at the code, there is no itemlen of 12, only up to 9, so it could be a safety thing too.

Reply:
Ah okay. Can you add something to your comment here then like "the documentation says" or "a comment in the function says" or something like that? Just so it's clear where that idea is coming from.